docs/DFAnumerics.rst

```diff
@@ -495,53 +495,91 @@ number of particles per kernel radius exceeds a given value :math:`n_{PPK}^{max}
 equations for the DFA were derived and all first order or smaller terms where neglected.
 Among those terms is the lateral shear stress. This term leads toward
 the homogenization of the velocity field. It means that two neighbor elements
 of fluid should have similar velocities. The aim behind adding artificial viscosity is to
 take this phenomena into account. The following viscosity force is added:
 
-
 .. math::
     \begin{aligned}
-    \mathbf{F_k^{viscosity}} = &- \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k\|^2 A_{Lat}
-    \frac{\mathbf{du}_k}{\|\mathbf{du}_k\|}\\
-    = & - \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k\| A_{Lat} \mathbf{du}_k
+    \mathbf{F_{viscosity}} = &- \frac{1}{2}\rho C_{Lat}\|\mathbf{du}\|^2 A_{Lat}
+    \frac{\mathbf{du}}{\|\mathbf{du}\|}\\
+    = & - \frac{1}{2}\rho C_{Lat}\|\mathbf{du}\| A_{Lat} \mathbf{du}
     \end{aligned}
 
-Where the velocity difference reads :math:`\mathbf{du}_k = \mathbf{u}_k - \mathbf{\bar{u}}`
+Where the velocity difference reads :math:`\mathbf{du} = \mathbf{u} - \mathbf{\bar{u}}`
 (:math:`\mathbf{\bar{u}}` is the mesh velocity interpolated at the particle position).
 :math:`C_{Lat}` is a coefficient that rules the viscous force. It would be the
 equivalent of :math:`C_{Drag}` in the case of the drag force. The :math:`C_{Lat}`
-is a numerical parameter that depends on the mesh size. Its value is set in the configuration (``subgridMixingFactor``) to 100
-by default (ToDo: and should be discussed and further tested).
+is a numerical parameter that depends on the mesh size. Its value is set to 100
+and should be discussed and further tested.
 
 Adding the viscous force
-~~~~~~~~~~~~~~~~~~~~~~~~~
+""""""""""""""""""""""""
 
 The viscous force is added implicitly:
 
 .. math::
   \begin{aligned}
-  \mathbf{F_k^{viscosity}} = &-\frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k^{old}\| A_{Lat}
-  \mathbf{du}_k^{new}\\
-  = &  -\frac{1}{2}\rho C_{Lat}\|\mathbf{u}_k^{old} - \mathbf{\bar{u}}^{old}\| A_{Lat}
-  (\mathbf{u}_k^{new} - \mathbf{\bar{u}}^{old})
+  \mathbf{F_{viscosity}} = &-\frac{1}{2}\rho C_{Lat}\|\mathbf{du}^{old}\| A_{Lat}
+  \mathbf{du}^{new}\\
+  = &  -\frac{1}{2}\rho C_{Lat}\|\mathbf{u}^{old} - \mathbf{\bar{u}}^{old}\| A_{Lat}
+  (\mathbf{u}^{new} - \mathbf{\bar{u}}^{old})
   \end{aligned}
-
 Updating the velocity is done in two steps. First adding the explcit term related to the
 mean mesh velocity and then the implicit term which leads to:
 
 .. math::
-  \mathbf{u}_k^{new} = \frac{\mathbf{u}_k^{old} - C_{vis}\mathbf{\bar{u}}^{old}}{1 + C_{vis}}
+  \mathbf{u}^{new} = \frac{\mathbf{u}^{old} - C_{vis}\mathbf{\bar{u}}^{old}}{1 + C_{vis}}
+
+With :math:`C_{vis} = \frac{1}{2}\rho C_{Lat}\|\mathbf{du}^{old}\| A_{Lat}\frac{dt}{m}`
+
+
+Ata Artificial viscosity
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-With :math:`C_{vis} = \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k^{old}\| A_{Lat}\frac{dt}{m}`
+An up-winding method based on Lax-Friedrichs scheme
```

Comment:
Is 'up-winding' a technical term here?

Reply:
yes
https://en.wikipedia.org/wiki/Upwind_scheme

Reply:
ah, then you should write 
`An upwind method based` -> related to the wind 
since up-winding is easily confused with the winding of e.g. a string or rope
https://www.dictionary.com/browse/winding